INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:7000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 15:39:38] "[33mGET / HTTP/1.1[0m" 404 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 15:39:38] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 15:39:42] "[31m[1mGET /login HTTP/1.1[0m" 405 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 15:40:15] "[35m[1mPOST /login HTTP/1.1[0m" 505 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:07:53] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:07:57] "[31m[1mPOST /login HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:08:01] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:08:02] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:09:38] "[33mGET /logged_user HTTP/1.1[0m" 404 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:10:05] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:11:05] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:11:10] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
ERROR:app:Exception on /logged_user [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 154, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/marshall/git_repos/_api_/marshall-api/app.py", line 51, in logged_user
    current_user = get_jwt_identity()
NameError: name 'get_jwt_identity' is not defined
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:11:49] "[35m[1mGET /logged_user HTTP/1.1[0m" 500 -
ERROR:app:Exception on /logged_user [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 154, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/marshall/git_repos/_api_/marshall-api/app.py", line 51, in logged_user
    current_user = get_jwt_identity()
NameError: name 'get_jwt_identity' is not defined
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:12:41] "[35m[1mGET /logged_user HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:12:54] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:13:10] "[31m[1mGET /logged_user HTTP/1.1[0m" 422 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:13:21] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
ERROR:app:Exception on /logged_user [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 154, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/marshall/git_repos/_api_/marshall-api/app.py", line 51, in logged_user
    current_user = get_jwt_identity()
NameError: name 'get_jwt_identity' is not defined
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:13:59] "[35m[1mGET /logged_user HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /logged_user [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 154, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/marshall/git_repos/_api_/marshall-api/app.py", line 51, in logged_user
    current_user = get_jwt_identity()
NameError: name 'get_jwt_identity' is not defined
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:15:05] "[35m[1mGET /logged_user HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:17:01] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
ERROR:app:Exception on /logged_user [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 154, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/marshall/git_repos/_api_/marshall-api/app.py", line 51, in logged_user
    current_user = get_jwt_identity()
NameError: name 'get_jwt_identity' is not defined
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:17:22] "[35m[1mGET /logged_user HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /logged_user [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'logged_user' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:17:58] "[35m[1mGET /logged_user HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:18:29] "GET /logged_user HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:18:35] "[31m[1mGET /logged_user HTTP/1.1[0m" 422 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:18:40] "GET /logged_user HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:22:43] "[35m[1mPOST /login HTTP/1.1[0m" 505 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:23:08] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:23:20] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:23:21] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:24:37] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:24:40] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:24:50] "GET /logged_user HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:24:52] "GET /logged_user HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:25:18] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:25:27] "GET /logged_user HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:29:21] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:31:44] "[31m[1mGET /refresh HTTP/1.1[0m" 405 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:32:05] "[31m[1mPOST /refresh HTTP/1.1[0m" 401 -
ERROR:app:Exception on /refresh [POST]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 154, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/marshall/git_repos/_api_/marshall-api/app.py", line 57, in refresh
    access_token = create_access_token(identity=identity, fresh=False)
NameError: name 'create_access_token' is not defined
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:32:16] "[35m[1mPOST /refresh HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:32:38] "POST /refresh HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:32:45] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:32:48] "GET /logged_user HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:40:07] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:40:18] "GET /logged_user HTTP/1.1" 200 -
ERROR:app:Exception on /logout [DELETE]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 154, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/marshall/git_repos/_api_/marshall-api/app.py", line 87, in logout
    token = get_jwt()
NameError: name 'get_jwt' is not defined
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:40:41] "[35m[1mDELETE /logout HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:41:21] "[31m[1mDELETE /logout HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:41:27] "POST /login HTTP/1.1" 200 -
ERROR:app:Exception on /logout [DELETE]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'logout' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:41:38] "[35m[1mDELETE /logout HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /logout [DELETE]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'logout' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:41:51] "[35m[1mDELETE /logout HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:42:35] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:42:43] "DELETE /logout HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:42:54] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:42:55] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:43:10] "POST /refresh HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:43:19] "GET /logged_user HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:43:36] "DELETE /logout HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:43:42] "[31m[1mPOST /refresh HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [12/Aug/2025 16:43:43] "[31m[1mPOST /refresh HTTP/1.1[0m" 401 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 09:59:16] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 09:59:18] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 09:59:19] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 09:59:20] "POST /login HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 09:59:24] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 09:59:25] "[31m[1mGET /logged_user HTTP/1.1[0m" 401 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 106, in getTransients
    print(dict(request.json))
TypeError: 'NoneType' object is not iterable
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 10:58:59] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 10:59:25] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 10:59:38] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 10:59:57] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:02:54] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "app.py", line 108, in getTransients
    models = models_transients_get(log, request.json, db=dbConn)
TypeError: 'module' object is not callable
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:24:46] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:25:06] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:26:08] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:26:46] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:27:56] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:30:56] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:32:16] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:32:33] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:33:07] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:34:32] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:35:17] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:36:05] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:36:37] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:36:52] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:37:54] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:38:20] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:38:44] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:39:24] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:40:14] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:40:45] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:41:15] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:41:46] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:42:25] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:42:42] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:43:12] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:44:49] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:45:03] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:46:33] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:46:34] "GET /getTransients HTTP/1.1" 200 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:46:54] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:47:42] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:48:40] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:51:20] "GET /getTransients HTTP/1.1" 200 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:51:24] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:51:48] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:51:59] "GET /getTransients HTTP/1.1" 200 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:52:04] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:52:10] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:53:01] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:54:10] "GET /getTransients HTTP/1.1" 200 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:54:23] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'and t.transientBucketId = p.transientBucketId   order by case when t.dateAdde...' at line 1")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p   and t.transientBucketId = p.transientBucketId   order by case when t.dateAdded is null then 1 else 0 end,  t.dateAdded desc
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:54:32] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:55:30] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:55:38] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:55:44] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:56:04] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:56:08] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:56:13] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:56:18] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:56:53] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:56:58] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:57:46] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:57:51] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:57:58] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 11:58:56] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:23:48] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:24:33] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:__main__:could not create the database cursor: 'str' object has no attribute 'cursor'
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:25:16] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:__main__:could not create the database cursor: 'NoneType' object has no attribute 'cursor'
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:25:56] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:__main__:could not create the database cursor: 'NoneType' object has no attribute 'cursor'
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:26:09] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:26:52] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:27:26] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:27:47] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:28:33] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 12:29:05] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:52:16] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:55:35] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:55:48] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:55:57] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:56:31] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:57:15] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:57:38] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:59:07] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:59:17] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:59:25] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 13:59:54] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:00:11] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:01:26] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:01:44] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:02:25] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:02:41] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:03:53] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:06:23] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:09:45] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:12:10] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:12:44] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:13:16] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:13:38] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:14:28] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:14:40] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:15:25] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:17:04] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:19:32] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:24:38] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:26:21] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:26:34] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:26:38] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:27:48] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:29:07] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:31:16] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:31:52] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:33:59] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:36:33] "GET /getTransients HTTP/1.1" 200 -
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:38:05] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:38:10] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:40:25] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:41:21] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:41:51] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:42:17] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:42:27] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:42:39] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:43:08] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:44:18] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:45:52] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:47:51] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:55:30] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 14:56:01] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:02:02] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:04:40] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:05:41] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:08:43] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:12:31] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:12:55] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:15:00] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:19:28] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:20:48] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:22:03] "GET /getTransients HTTP/1.1" 200 -
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:22:16] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:22:25] "GET /getTransients HTTP/1.1" 200 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1054, "Unknown column 't.observationDate' in 'ORDER BY'")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p  where  1=1 and t.`decDeg` < 30  and t.transientBucketId = p.transientBucketId   order by case when t.observationDate is null then 1 else 0 end,  t.observationDate 
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:24:41] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
WARNING:__main__:MySQL raised an error - read command not executed.
(1054, "Unknown column 't.observationDate' in 'ORDER BY'")
Here is the sqlQuery
	
                    select t.transientBucketId from transientBucketSummaries t, pesstoObjects p  where  1=1 and t.`decDeg` < 30  and t.transientBucketId = p.transientBucketId   order by case when t.observationDate is null then 1 else 0 end,  t.observationDate 
                 limit 0, 10
ERROR:app:Exception on /getTransients [GET]
Traceback (most recent call last):
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/home/marshall/Anaconda3-2023.09-0-Linux-x86_64/envs/marshall-api/lib/python3.6/site-packages/flask/app.py", line 1702, in make_response
    f"The view function for {request.endpoint!r} did not"
TypeError: The view function for 'getTransients' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:24:53] "[35m[1mGET /getTransients HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:25:09] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:26:01] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:26:51] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:28:37] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:30:13] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:30:32] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [18/Aug/2025 15:39:24] "GET /getTransients HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:8000/ (Press CTRL+C to quit)
